### PR TITLE
ns5x_tgl: Add release v1.5.2

### PR DESCRIPTION
**Pull request by mkopec**

### CODE REVIEW

File: docs/variants/novacustom_ns5x_tgl/releases.md [hunk 1]
```diff
@@ -13,6 +13,85 @@ For details about our release process please read
...
+- [Function Lock setting is not saved after cold boot](https://github.com/Dasharo/dasharo-issues/issues/458)
+- [No ability to change active PCR banks with TPM PPI in FW](https://github.com/Dasharo/dasharo-issues/issues/521)
+- [Turning wifi/bt off inside BIOS leads to an ACPI error message while system start on Gentoo Linux](https://github.com/Dasharo/dasharo-issues/issues/638)
+- [The laptop does not automatically wake up from hibernation using rtcwake](https://github.com/Dasharo/dasharo-issues/issues/485)
+- [Power state after power failure option does not work as intended](https://github.com/Dasharo/dasharo-issues/issues/524)
+- [OS fails to resume from S3 on some board revisions](https://github.com/Dasharo/dasharo-issues/issues/523)
+- [Windows 11 fails to resume from hibernation](https://github.com/Dasharo/dasharo-issues/issues/529)
+- [No HDMI output in firmware](https://github.com/Dasharo/dasharo-issues/issues/533)
+- [Connecting and immediately disconnecting the charger, sets the battery status in OS to "charging" for about 2 minutes](https://github.com/Dasharo/dasharo-issues/issues/350)
```
Comment (miczyg1) [inline docs/variants/novacustom_ns5x_tgl/releases.md:66]: I'm surprised pre-commit did not yell about 80 chars limit... I had to wrap the lines for relnotes...

**Review by miczyg1 — APPROVED** [review]

File: docs/variants/novacustom_ns5x_tgl/releases.md (hunk 1 above)
Comment (mkopec) [reply docs/variants/novacustom_ns5x_tgl/releases.md:66]: I it doesn't complain if the overflow is a link ¯\_(ツ)_/¯

File: docs/variants/novacustom_ns5x_tgl/releases.md [hunk 2]
```diff
@@ -13,6 +13,85 @@ For details about our release process please read
...
+- [EC applying power limits may sometimes fail](https://github.com/Dasharo/dasharo-issues/issues/625)
+- [Wrong power limits are applied on AC attach / detach](https://github.com/Dasharo/dasharo-issues/issues/626)
+- [USB-PD controller may hang in glitched state on shutdown](https://github.com/Dasharo/dasharo-issues/issues/627)
+- [Yellow bangs in device manager](https://docs.dasharo.com/unified/clevo/post-install/#unrecognized-usb-controller-device-in-device-manager)
+- [Random freezes in Windows 11](https://github.com/Dasharo/dasharo-issues/issues/628)
+- [Low level interfering crackling/popping of the speakers while playing no sound](https://github.com/Dasharo/dasharo-issues/issues/224)
+- [ACPI error during boot (system76_acpi::kbd_backlight)](https://github.com/Dasharo/dasharo-issues/issues/559)
+- [Thunderbolt may not work after wakeup from sleep](https://github.com/Dasharo/dasharo-issues/issues/629)
+- [Unnecessary security warning after firmware update](https://github.com/Dasharo/dasharo-issues/issues/556)
+- [Early boot DMA protection setting in UEFI does not work correctly](https://github.com/Dasharo/dasharo-issues/issues/630)
```
Comment (BeataZdunczyk) [inline docs/variants/novacustom_ns5x_tgl/releases.md:50]: @mkopec fixed or know issue? https://github.com/Dasharo/dasharo-issues/issues/548#issuecomment-1878785880

**Review by BeataZdunczyk — CHANGES_REQUESTED** [review]

File: docs/variants/novacustom_ns5x_tgl/releases.md (hunk 2 above)
Comment (mkopec) [reply docs/variants/novacustom_ns5x_tgl/releases.md]: @BeataZdunczyk right, moved to known issues for ns5x and nv4x tgl: https://github.com/Dasharo/docs/pull/739/commits/2010062bfa8cac881e3fc0bdf65f53c71e4f9ae0

**Pull request merged**